gui/dist_view/distviewcompute.h

```diff
@@ -30,7 +30,7 @@
  * hash key (the hash key is not part of the Bin class
  */
 struct Bin {
-	Bin() : weight(0.f) {}
+	Bin() : weight(0.f), color(0.f, 0.f, 0.f) {}
 	Bin(const multi_img::Pixel& initial_means)
 		: weight(1.f), means(initial_means) {} //, points(initial_means.size()) {}
```

Comment:
the constructor takes initial means, I suppose it should also take an initial color mix. can you check where it is called?